docs/conf.py

```diff
@@ -11,10 +11,12 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-import sys
 import os
+import re
+import sys
 
 from django import __version__ as dj_version_actual
+from pootle.core.utils import version as pootle_version
 from translate.__version__ import sver as ttk_version_actual
```

Comment:
Since imports are being rearranged it should be:

```python
import os
import re
import sys

from translate.__version__ import sver as ttk_version_actual

from django import __version__ as dj_version_actual

from pootle.core.utils import version as pootle_version
```